### ISSUE TITLE
[Important] SATT now on WooCommerce.com

### ISSUE THREAD

**Issue by manospsyx** [Prospress/woocommerce-subscribe-all-the-things#436]
### TL;DR

SATT is now available on WooCommerce.com under a new name: [All Products For WooCommerce Subscriptions](https://woocommerce.com/products/all-products-for-woocommerce-subscriptions/?aff=46147&cid=5699243) (APFS). APFS is released, maintained and supported by [SomewhereWarm](https://somewherewarm.gr), the team that has led the development of SATT since 2015.

### The Story

It's 2015 and Subscriptions v2.0 is almost ready for release, when, one day, an unusual feature request for Product Bundles arrives in my mailbox: A store owner looking for a way to offer subscription boxes is asking if a "Subscription Bundle" product type could be included in a future release. 

This feature request struck a chord with me. I had an idea about what it would take to do this right, but there was no way I could pull this off alone. I decided to run it by @thenbrent at Prospress, to see what he thought. As it turned out, Prospress had already logged a fair amount of similar requests, and both I and Brent had identified the same bottleneck -- a strong coupling between:

* the ability to purchase a product on subscription, and 
* its product type. 

This limitation prevented store owners from offering existing products on subscription -- for example, some store owners were interested in letting customers subscribe to entire carts.

It didn't take us long to agree that the problem we were looking at was well worth taking a shot at. A few weeks after deciding to look into it, we had a solution, and SATT v1 was born.

For a long time, SATT was only a side-project, labelled as "experimental/beta" as we evaluated the possibility of rolling it into a future Subscriptions release. During this time, we actively discouraged store owners and developers from becoming dependent on it: We couldn't officially support a project with such an unclear future.

As the project matured, it became evident to us that:

* There is technically no feasible, backwards compatible way to make SATT's core concepts the new standard for WooCommerce Subscriptions.
* Subscription-type products are here to stay. Both approaches are needed and always will be.

Including both approaches in WC Subscriptions is something we evaluated, but eventually rejected as potentially confusing. Think about it: When should store owners use subscription-type products and when should they resort to subscription plans? Fundamental user decisions like this shouldn't be hard to make, or to explain.

As new features were added to SATT, its user base kept growing and new feature requests kept coming in. However, our involvement with it did not follow the same pattern. We realized that, to help more store owners and developers succeed with SATT, Subscriptions and WooCommerce, we needed to figure out a way to:

* Maintain and support it on a daily basis.
* Prioritize new features fairly and effectively.
* Allocate resources to its development in a data-driven manner.
* Increase the visibility of the project in the WooCommerce ecosystem.

To make all this possible, we decided that the best way forward for SATT was to:

1. Assign all development, maintenance and support duties to a single team. 
2. Commercialize it.

### Now What?

Today, I am happy to let you all know that SATT will be soon available on WooCommerce.com under a new name: [All Products For WooCommerce Subscriptions](https://woocommerce.com/products/all-products-for-woocommerce-subscriptions/?aff=46147&cid=5699243) (APFS). The new extension will be released, maintained and supported by SomewhereWarm. 

**The core functionality and codebase of APFS is identical to SATT**. APFS simply continues where SATT left off. To make this as clear as possible, APFS will launch at version **2.2** -- up a notch from the latest minor version of SATT, **2.1**.

Here's what to expect in APFS:

1. A re-designed admin UI.
2. [Cart subscription plan discounts](https://github.com/somewherewarm/woocommerce-subscribe-all-the-things/issues/57).

### What's Next?

If you are currently using SATT on a production site, please be aware that:

1. The latest version of SATT is **2.1.5**. It has been tested to work with WooCommerce **v3.0-3.5** and WooCommerce Subscriptions **v2.1-2.5**.
2. To get the latest features and bug fixes, please upgrade to [All Products For WooCommerce Subscriptions](https://woocommerce.com/products/all-products-for-woocommerce-subscriptions/?aff=46147&cid=5699243).

If you decide to upgrade SATT to APFS:

1. **De-activate** and **Delete** SATT. It's okay to delete the plugin -- all data will remain intact.
2. [Get](https://woocommerce.com/products/all-products-for-woocommerce-subscriptions/?aff=46147&cid=5699243) APFS.
3. **Upload**, **Install** and **Activate** it.

The release of **All Products For Subscriptions** on WooCommerce.com breathes new life into SATT, as it finally brings the project out of its "side-project" limbo:

* WooCommerce store owners who currently use SATT can count on a constant stream of new features, updates and fixes in the future.
* WooCommerce developers and store builders can safely use it in their projects, knowing that the functionality is here to stay.

As you may have noticed, the SATT repository has been transferred to the SomewhereWarm organization on GitHub, and it **will soon be archived** to indicate that **SATT is no longer maintained**. Archiving the repository will basically "lock it" to its current state.

I'd like to take this opportunity to say a big **Thank You to all SATT contributors** who helped move the project forward. If you have contributed code, issues, or ideas to SATT, please follow APFS at its new home and help us make it better. To get access to the APFS repository, please send a request to `gh [at] somewherewarm.gr`.

Got questions, or thoughts you'd like to share? You know [where to find us](https://somewherewarm.gr/about/)!
